easybuild/easyconfigs/a/ARP_wARP/ARP_wARP-7.6.eb

```diff
@@ -0,0 +1,78 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Mazur
+# Research IT
+# Biozentrum, University of Basel
+# Modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock='Tarball'
+
+name = 'ARP_wARP'
+version = '7.6'
+
+homepage = 'http://www.embl-hamburg.de/ARP/'
+description = """ Crystallographic macromolecular model building. Builds proteins, RNA/DNA, 
+ secondary structure, side chains, loops, solvent and ligands. Intended to be used in 
+ conjunction with CCP4. """ 
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# Need to agree to the licence for the download, either from the CCP4 site or from
+# http://www.embl-hamburg.de/ARP/download_request7.6.html
+sources = ['%(namelower)s_%(version)s.tar.gz']
+
+ccp4dir = '%(namelower)s-%(version)s'
+arpwarpdir = 'arp_warp_7.6'
+
+postinstallcmds = [
+    'touch $HOME/.agree2ccp4v6',
```

Comment:
this is pretty nasty, especially without cleaning it up afterwards...

there's no flag for this?